test/bin/test_expect/dune.inc

```diff
@@ -59,6 +59,16 @@
   )
 )
 
+(alias
+  (name runtest)
+  (deps (:x tabs.md) (:y tabs.md.expected) (package mdx))
+  (action
+    (progn
+      (run ocaml-mdx test --force-output  %{x})
```

Comment:
The warnings are printed to stderr and are not checked and spam the test outputs.
Maybe after https://github.com/realworldocaml/mdx/pull/194 you could redirect both outputs to the output file ?